action.yml

```diff
@@ -12,6 +12,10 @@ inputs:
     description: 'Enter an existing Slot other than the Production slot'
     required: false
     default: 'production'

```

Comment:
can you try this approach https://www.npmjs.com/package/@actions/github#webhook-payload-typescript-definitions if it works

then you don't need to take it as input.

Reply:
taking input has any disadvantages? As we are passing default value, user need not set it.